### DISCUSSION TITLE
GUI V 1.2.11 asking passphrase I have never created...

### DISCUSSION THREAD

**Discussion by myCryptoUK** [Chia-Network/chia-blockchain#9153] [Support]
Hi,

I have upgraded my GUI to 1.2.11 and when I try to restart it says;

Your key ring is locked
Please enter your passphrase

I have never defined any passphrase. I checked the ".keyring.yaml" file and it is 0 kb.

Can anyone help me how to define a passphrase and use in this case..

Thanks.
CryptoUK...

**Comment by Jacek-ghub** [comment]
From which version have up updated?  Maybe v1.2.11 assumes that v1.2.9/10 already setup the keyring, and doesn't bother to do the initialization, as those two previous versions were doing.

**Comment by myCryptoUK** [comment]
I have updated from v 1.2.10. Is there way to set up the key ring now? In the previous two versions I haven't been asked to set up a keyring either. Otherwise I would do it...

Sent from Yahoo Mail on Android 
 
  On Fri, 5 Nov 2021 at 18:56, ***@***.***> wrote:   


From which version have up updated? Maybe v1.2.11 assumes that v1.2.9/10 already setup the keyring, and doesn't bother to do the initialization, as those two previous versions were doing.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by loppefaaret** [comment]
Close Chia - move `~\.chia_keys\keyring.yaml` out of the way - open Chia

**Comment by myCryptoUK** [comment]
Thank you very much... worked well...Via 


Sent from Yahoo Mail on Android 
 
  On Mon, 8 Nov 2021 at 13:45, ***@***.***> wrote:   


Close Chia - move ~\.chia_keys\keyring.yaml out of the way - open Chia

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
